src/map/status.cpp

```diff
@@ -15663,7 +15663,7 @@ AttributeDatabase elemental_attribute_db;
  * @param level Element level 1 ~ MAX_ELE_LEVEL
  */
 int16 AttributeDatabase::getAttribute(uint16 level, uint16 atk_ele, uint16 def_ele) {
-	if (!CHK_ELEMENT(atk_ele) || !CHK_ELEMENT(def_ele) || !CHK_ELEMENT_LEVEL(level+1))
+	if (!CHK_ELEMENT(atk_ele) || !CHK_ELEMENT(def_ele) || !CHK_ELEMENT_LEVEL(level))
 		return 100;
 
 	return this->attr_fix_table[level][atk_ele][def_ele];
```

Comment:
```suggestion
	return this->attr_fix_table[level-1][atk_ele][def_ele];
```
https://github.com/rathena/rathena/blob/master/src/map/status.hpp#L138

Reply:
Sorry about that